docs/reference/smb.rst

```diff
@@ -0,0 +1,5 @@
+Samba Plugin
```

Comment:
Samba -> SMB (SMB is the protocol, Samba is a Linux implementation of SMB)

Reply:
Whoops, thanks